pynicotine/core.py

```diff
@@ -271,8 +271,10 @@ def quit(self, signal_type=None, _frame=None, should_finish_uploads=False):
 
     def connect(self):
 
+        if self.user_status == slskmessages.UserStatus.ONLINE:
```

Comment:
Away is the same as online in this context.

`== slskmessages.UserStatus.ONLINE` -> `=! slskmessages.UserStatus.OFFLINE`